server/src/controllers/chess-game.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
+import CGUtil from '../utils/chess-game-util';
+
+const getRating = (req, res) => {
+  const { playerOneRating, playerTwoRating, winner } = req.query;
+  const newRating = CGUtil.calcRating(playerOneRating, playerTwoRating, winner);
+  res.json(newRating);
+};
+
+export default {
```

Comment:
why unnamed object export?
Also, this file has 
/* eslint-disable import/prefer-default-export */
and has a default export

Reply:
Our linter rules ([details](https://github.com/benmosher/eslint-plugin-import/blob/v2.17.2/docs/rules/prefer-default-export.md)) give an error if there is single export and it is not default.
forgot to remove eslint disable comment, thank for pointing out.